bin/templates/cordova/lib/check_reqs.js

```diff
@@ -194,11 +193,9 @@ module.exports.check_java = function () {
             }
         }
     }).then(function () {
-        return Q.denodeify(child_process.exec)('javac -version')
```

Comment:
The `Q` changes I think should done in the dropping Q PR : https://github.com/apache/cordova-android/pull/833, but this type of work kinda blends together.

Reply:
What I wanted to do in this PR is to ensure that all external process spawning would be done with `execa`. The fact that this also happens to remove `Q` usage is circumstantial. So I think this is alright to have here.

Also note that this PR is supposed to be merged before #833. Maybe then it makes more sense?